CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+# 1.2.8 (2021-02-24)
+## Edge Agent
+### Bug Fixes
+* Fix underflow possibility on ColumnFamilyDbStore  ( [bc78f1c](https://github.com/Azure/iotedge/commit/bc78f1c0b0ab0dfa305b817c030b82cea035f6e0) )
+* Remove BouncyCastle dependency ( [403ca87](https://github.com/Azure/iotedge/commit/403ca87bc228421c8913c1431687267dee9112a7), [7589457](https://github.com/Azure/iotedge/commit/7589457deec0f2735711cb8973cac0f0e17046c1) )
+* Remove Remove k8s projects ( [32e96d6](https://github.com/Azure/iotedge/commit/32e96d684157fe65904baf8b6beba4505a8200ba) )
```

Comment:
I think you can remove the k8s entry. Our k8s support is in preview, and is based in the 1.1 branch, so the work done here to remove k8s-related code doesn't actually impact the 1.2 product in any way.